### PR TITLE
grouper-ctl command to create service account

### PR DESCRIPTION
**Pull request by cauthu**
```
$ python grouper/ctl/main.py service_account -h 
usage: main.py service_account [-h] -a ACTOR_NAME {create} ...

positional arguments:
  {create}
    create              Create a new service account

optional arguments:
  -h, --help            show this help message and exit

required named arguments:
  -a ACTOR_NAME, --actor ACTOR_NAME
                        Name of the entity performing this action. Must be a
                        valid Grouper human or service account.
$
$ python grouper/ctl/main.py service_account create -h
usage: main.py service_account create [-h]
                                      name owner_group machine_set description

positional arguments:
  name         Name for the service account
  owner_group  Name of the owner group. Must be a valid Grouper group
  machine_set  The machine set for the service account
  description  Description for the service account

optional arguments:
  -h, --help   show this help message and exit
$
```

### CODE REVIEW

**Comment by cauthu** [comment]
manual testing
```
$ python grouper/ctl/main.py service_account --actor nouser@example.com create badname group-foo-bar 'machine_set_abc' 'description\nbar\nfoo'
2018-12-07 01:49:35,180 ERROR   Invalid service account name "badname"  [root]
$ 
$ python grouper/ctl/main.py service_account --actor nouser@example.com create good@a.b group-foo-bar 'machine_set_abc' 'description\nbar\nfoo'
2018-12-07 01:49:52,221 CRITICAL        Actor user "nouser@example.com" is not a valid Grouper user  [root]
$ 
$ python grouper/ctl/main.py service_account --actor cauthu@example.com create good@a.b group-foo-bar 'machine_set_abc' 'description\nbar\nfoo'
2018-12-07 01:50:12,759 CRITICAL        Owner group "group-foo-bar" does not exist.  [root]
$ 
$ python grouper/ctl/main.py service_account --actor cauthu@example.com create good@a.b group-foo-bar 'machine_set_abc' 'description\nbar\nfoo'
2018-12-07 01:50:59,184 INFO    good@a.b: No such service account, creating...  [root]
$ 
```

File: README.rst [hunk 1]
```diff
@@ -104,7 +104,23 @@ Once chromedriver is installed, the tests can be run using pytest:
 
 .. code:: bash
 
+    export PYTHONPATH=$(pwd)
```
Comment (cauthu) [inline README.rst]: i'm not sure why this is showing up. while in my branch, `git diff master..` does not show this change.

Comment (wrodriguez-dbx) [reply README.rst]: Yea not sure either -- if it messes anything up should be easy to clean up after

Comment (cbguder) [reply README.rst]: @cauthu Your base branch and local master probably don’t match upstream master.

Comment (cauthu) [reply README.rst]: i'll just merge and if it breaks i'll fix :)

Comment (cauthu) [reply README.rst]: @cbguder so i guess it merged properly, i.e., the readme.md change was not in the diff /shrug

**Pull request merged**